### PR TITLE
Update Select.tsx

### PR DESCRIPTION
**Pull request by AaronAgility**
Changing the option value to label for select component

### CODE REVIEW

**Review by DakotahGodfrey — APPROVED** [review]

**Comment by DakotahGodfrey** [comment]
Sorry can you also increment the version number in the package.json I'll publish the latest version once this has been merged.

**Review by DakotahGodfrey — DISMISSED** [review]

**Review by DakotahGodfrey — APPROVED** [review]

File: stories/molecules/inputs/select/Select.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ const Select: React.FC<ISelectProps> = ({
 			>
 				{options.map(({ value, label }) => {
 					return (
-						<option key={value} value={value}>
+						<option key={value} value={label}>
```
Comment (DerekAgility) [inline stories/molecules/inputs/select/Select.tsx:98]: @AaronAgility We have to revert this change that you made in this [PR](https://github.com/agility/agility-plenum-ui/pull/84) as it was causing all selects in the manager app to now work properly. Was there a reason for this change?
